test/handshake_helper.c

```diff
@@ -278,6 +279,15 @@ static void configure_handshake_ctx(SSL_CTX *server_ctx, SSL_CTX *server2_ctx,
     unsigned char *ticket_keys;
     size_t ticket_key_len;
 
+    TEST_check(SSL_CTX_set_max_send_fragment(server_ctx,
+                                             test->max_fragment_size));
```

Comment:
We use `!foo()` for boolean returns but since this is a positive check, I'll change it to `== 1` here to make it clearer.